fireapp/lib/presentation/volunteer_information/volunteer_information.dart

```diff
@@ -0,0 +1,204 @@
+                SizedBox(height: 12.0),
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      'Volunteer ID: ${volunteerInformation.ID}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Name: ${volunteerInformation.firstName} ${volunteerInformation.lastName}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Email: ${volunteerInformation.email}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Mobile No: ${volunteerInformation.mobileNo}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Preferred Hours: ${volunteerInformation.prefHours}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Experience Years: ${volunteerInformation.expYears}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text('Volunteer Possible Roles: ${volunteerInformation.possibleRoles.join(", ")}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Qualifications: ${volunteerInformation.qualifications.join(", ")}',
```

Comment:
qualifications is a List<Qualification>. If you simply join them as a string, flutter will call <instance of qualification>.toString(), resulting in formatting like `Qualification(id: 1, name: Stuff, updated: ..., created: ...)`.

Do not show the information like this. In this case, you should just show the name field of Qualification, and then separate them like I suggested above.